File: Placements.DataAccess/placement/Models/Trainerschedule.cs
using System;
using System.Collections.Generic;

namespace Placements.DataAccess.placement.Models;

public partial class Trainerschedule
{
    public long Id { get; set; }

    public long? CompanyId { get; set; }

    public long? SchoolId { get; set; }

    public long? CourseId { get; set; }

    public DateTime? StartDate { get; set; }

    public DateTime? EndDate { get; set; }

    public long? TrainerId { get; set; }

    public string? ScheduleType { get; set; }

    public long? StudentId { get; set; }

    public ulong IsDeleted { get; set; }

    public virtual Trainingcourse? Course { get; set; }

    public virtual Trainer? Trainer { get; set; }
}

File: web-app/src/app/services/types/Trainerschedule.ts
import { Trainer } from "./Trainer";
import { Trainingcourse } from "./Trainingcourse";

export type Trainerschedule = {
    Id: number;
    CompanyId?: number;
    SchoolId?: number;
    CourseId?: number;
    StartDate?: Date;
    EndDate?: Date;
    TrainerId?: number;
    ScheduleType?: string;
    StudentId?: number;
    IsDeleted: boolean;
    Course: Trainingcourse[];
    Trainer: Trainer[];
};
